contracts/ColonyNetwork.sol

```diff
@@ -124,16 +124,31 @@ contract ColonyNetwork is ColonyNetworkStorage {
     emit MetaColonyCreated(metaColony, _tokenAddress, skillCount);
   }
 
+  function createColony(address _tokenAddress, uint256 _version) public
+  stoppable
+  returns (address)
+  {
+    require(colonyVersionResolver[_version] != address(0x00), "colony-network-invalid-version");
+    return createColonyFunctionality(_tokenAddress, _version);
+  }
+
   function createColony(address _tokenAddress) public
   stoppable
   returns (address)
   {
     require(currentColonyVersion > 0, "colony-network-not-initialised-cannot-create-colony");
+    return createColonyFunctionality(_tokenAddress, currentColonyVersion);
+  }
+
+  function createColonyFunctionality(address _tokenAddress, uint256 _version) internal
```

Comment:
I don't like this name, it's too abstract -- can we use `createColonyFromVersion` instead?